examples/pytorch.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: jobset.x-k8s.io/v1alpha1
```

Comment:
can we create a pytorch folder, I think other examples may require more than one artifact, so better to put each example in a different folder. 

so something like:
examples/pytorch/resnet.yaml

Reply:
done